nano/secure/ledger.cpp

```diff
@@ -787,6 +785,9 @@ void nano::ledger::initialize (nano::generate_cache const & generate_cache_a)
 			this->cache.cemented_count += cemented_count_l;
 		});
 	}
+
+	auto transaction (store.tx_begin_read ());
+	cache.pruned_count = store.pruned_count (transaction);
```

Comment:
Cache initialization isn't tested, could add it to `ledger.cache` test maybe?

Reply:
Updated with test